zdict/dictionaries/apc.py

```diff
@@ -0,0 +1,172 @@
+import random
```

Comment:
please sort these `import` statement in alphabet order

Reply:
Updated